services/rewards/service.go

```diff
@@ -103,30 +111,44 @@ func (s *Service) GetParameters(ctx context.Context, currency *BaseCurrency) (*P
 
 	var rate, _ = rateData.Payload[currencyStr].Float64()
 
-	// merge in static s3 attributes into response
-	var (
-		payoutStatus     *custodian.PayoutStatus
-		custodianRegions *custodian.Regions
-		bucket, ok       = ctx.Value(appctx.ParametersMergeBucketCTXKey).(string)
-	)
-	logger.Debug().Str("bucket", bucket).Msg("merge bucket env var")
-	if ok {
-		// get payout status
-		logger.Debug().Str("bucket", bucket).Msg("extracting payout status")
-		payoutStatus, err = custodian.ExtractPayoutStatus(ctx, s.s3Client, bucket)
-		if err != nil {
-			return nil, fmt.Errorf("failed to get payout status parameters: %w", err)
-		}
-		logger.Debug().Str("bucket", bucket).Str("payout status", fmt.Sprintf("%+v", *payoutStatus)).Msg("payout status")
-
-		// get the custodian regions
-		logger.Debug().Str("bucket", bucket).Msg("extracting custodian regions")
-		custodianRegions, err = custodian.ExtractCustodianRegions(ctx, s.s3Client, bucket)
-		if err != nil {
-			return nil, fmt.Errorf("failed to get custodian regions parameters: %w", err)
+	s.cacheMu.RLock()
+	lastPollTime := s.lastPollTime
```

Comment:
Might be missing it somewhere but does `s.lastPollTime` need to be updated in the if block below if after 15 mins

Reply:
https://github.com/brave-intl/bat-go/pull/1700/commits/2af7238e0746cb783ca31b68ea9bf5ceebcfb6c5